sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -1,13 +1,13 @@
 # Azure Form Recognizer client library for Python
 
-Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to extract text and table data
+Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to recognize text and table data
 from form documents. Form Recognizer is made up of the following services:
 
-* Custom models - Extract name/value pairs and table data from forms. These models are trained with your own data, so they're tailored to your forms.
-* Prebuilt receipt model - Extract data from USA sales receipts using a prebuilt model.
-* Layout API - Extract text and table structures, along with their bounding box coordinates, from documents.
+* Prebuilt receipt model - Recognize data from USA sales receipts using a prebuilt model.
+* Custom models - Recognize name/value pairs and table data from forms. These models are trained with your own data, so they're tailored to your forms. You can then take these custom models and recognize forms. You can also manage the custom models you've created and see how close you are to the limit of custom models your account can hold.
+* Content API - Recognize text and table structures, along with their bounding box coordinates, from documents.
```

Comment:
For text analytics we kept this section consistent with the service. e.g. used terminology "Named Entities" instead of "Categorized Entities". If we say "Content API", I think we need to at least mention something about Layout of form, etc so people can make that 1:1 mapping

Reply:
Makes sense, I'm changing it back to Layout API